src/pages/core/plugins/transfer-delegate.md

```
@@ -93,3 +112,184 @@ pub async fn add_transfer_delegate_plugin() {
+    .asset(ctx.accounts.asset)
+    .collection(Some(ctx.accounts.collection))
+    .authority(Some(ctx.accounts.authority))
+    .payer(ctx.accounts.payer)
+    .plugin_type(PluginType::TransferDelegate)
+    .new_authority(PluginAuthority::Address { address: ctx.accounts.new_authority.key() })
+    .invoke()?;
+```
+{% /dialect %}
+
+{% dialect title="Rust" id="rust" %}
+```rust
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)